
object_detector:
  ros__parameters:
    # tuning_mode: false # Could set this here but leave it off so it can be easily set by the launch script
    x_min: 0
    x_max: 100
    y_min: 0
    y_max: 100
    h_min: 100
    h_max: 140
    s_min: 126
    s_max: 255
    v_min: 0
    v_max: 255
    sz_min: 0
    sz_max: 100

object_follower:
  ros__parameters:
    rcv_timeout_secs: 1.0
    angular_chase_multiplier: 0.7
    forward_chase_speed: 0.1
    search_angular_speed: 0.5
    max_size_thresh: 0.1
    filter_value: 0.9
    kp: 1
    ki: 0.1
    kd: 0.25